### ISSUE TITLE
lost text in blockquote when change to wysiwyg mode

### ISSUE THREAD

**Issue by phap96019** [nhn/tui.editor#1756]
## When I insert this text in markdown mode, then change to wysiwyg mode. I lost my content in blockquote

```
### ABC ABC
1. Goal

	> This text will be lost
```

## Screenshots
![image](https://user-images.githubusercontent.com/34448869/129658908-bdbdfd7a-5c99-46ec-9371-904140de4d60.png)
![image](https://user-images.githubusercontent.com/34448869/129658942-e3813eb7-3e29-4fc7-a3bb-4e8572e5ab09.png)

## Desktop:
 - OS: Ubuntu
 - Browser Chrome.

**Comment by ats1999** [comment]
```js 
### ABC ABC

1. Goal

> This text will be lost
```

This will not lost.

**Comment by ats1999** [comment]
@phap96019 
Can you give some information about this editor: 
  https://docs.google.com/forms/d/e/1FAIpQLSdhPRNZlEknQcx0V9Ww5U2sAbN_wv3WTmFeWIOIX21LRjFd7Q/viewform

**Comment by phap96019** [comment]
> ```js
> ### ABC ABC
> 
> 1. Goal
> 
> > This text will be lost
> ```
> 
> This will not lost.

I know that, but my boss asked it to work with the above text.

**Comment by ats1999** [comment]
> > ```js
> > ### ABC ABC
> > 
> > 1. Goal
> > 
> > > This text will be lost
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > This will not lost.
> 
> I know that, but my boss asked it to work with the above text.

Some one from team will surely help you!

**Comment by js87zz** [comment]
@phap96019 
I checked the issue. 
I will fix it and comment here. Thanks!

**Comment by js87zz** [comment]
@phap96019 
The bug has been fixed in `v3.1.0`.

**Comment by phap96019** [comment]
> @phap96019
> The bug has been fixed in `v3.1.0`.

thank you!!
